script.js

```diff
@@ -9,3 +9,25 @@ document.querySelectorAll(".toggleButton").forEach((button) => {
     }
   });
 });
+
+let operand1 = 1;
+const output = document.querySelector('[data-js="like-output"]');
+
+const increaseByOneButton = document.querySelector(
+  '[data-js="increase-likes"]'
+);
+
+increaseByOneButton.addEventListener("click", () => {
+  let increaseByOneButton = operand1++;
+
+  output.textContent = increaseByOneButton;
+  console.log(increaseByOneButton);
+});
+
+const bodyElement = document.querySelector('[data-js="body"]');
+
+const toggleModeButton = document.querySelector('[data-js="toggle-button"]');
+toggleModeButton.addEventListener("click", () => {
+  bodyElement.classList.toggle("dark");
+});
+console.log(toggleModeButton);
```

Comment:
since this console.log is outside of the Eventlistener, it will only printed once at the page load.